switchmap/server/db/table/metrics.py

```
@@ -0,0 +1,89 @@
+        return int(datetime.datetime.now(datetime.timezone.utc).timestamp())
+    if isinstance(value, (int, float)):
+        return int(value)
+    if isinstance(value, datetime.datetime):
+        return int(value.timestamp())
+    if isinstance(value, str):
+        try:
+            return int(datetime.datetime.fromisoformat(value).timestamp())
+        except ValueError as exc:
+            raise TypeError(
+                "Invalid ISO-8601 datetime string for last_polled"
+            ) from exc
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)